### PR TITLE
Add EBU BISS-CA

### PR DESCRIPTION
**Pull request by funman**
RFC for emm_set_length and emm_validate

### CODE REVIEW

**Comment by cmassiot** [comment]
I am not sure the prefix emm_ is adequate here, because BISS-CA is just one type of EMM. There are many other types of EMM.

**Comment by funman** [comment]
I can make it bissca_emm

**Comment by cmassiot** [comment]
I think it would be more future-proof.

**Comment by funman** [comment]
Prefixed the functions, there are still a number of issues though, maybe they can make more sense when compared with their usage

File: ebu/biss.h [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+{
+#endif
+
+/*****************************************************************************
+ * Conditional Access Table
+ *****************************************************************************/
+#define EMM_TABLE_ID            0x81
+#define EMM_HEADER_SIZE         PSI_HEADER_SIZE_SYNTAX1
+
+#define EMM_CIPHER_RSA_2048_OAEP    0
```
Comment (cmassiot) [inline ebu/biss.h]: I think you need to prefix these too.

**Pull request merged**